br/pkg/restore/client.go

```diff
@@ -754,6 +754,7 @@ func (rc *Client) switchTiKVMode(ctx context.Context, mode import_sstpb.SwitchMo
 			gctx,
 			store.GetAddress(),
 			opt,
+			grpc.WithBlock(),
```

Comment:
should we also include [`grpc.FailOnNonTempDialError(true)`](https://pkg.go.dev/google.golang.org/grpc@v1.29.0#FailOnNonTempDialError)